pkg/controller/controller_ref.go

```diff
@@ -42,9 +42,21 @@ func GetControllerOf(pod *k8sv1.Pod) *metav1.OwnerReference {
 	return nil
 }
 
+// IsControlledBy determine a pod is controlled by a given VirtualMahcineInstance.
+//
+// Currenly, the following kinds of pods are supposed to be controlled by a VMI:
+//
+// - the main virt-launcher pod associated with the VMI
+// - the companion hotplug volume attachment pod
 func IsControlledBy(pod *k8sv1.Pod, vmi *virtv1.VirtualMachineInstance) bool {
 	if controllerRef := GetControllerOf(pod); controllerRef != nil {
 		return controllerRef.UID == vmi.UID
 	}
 	return false
 }
+
+// IsAttachmentPod determine whether a given pod is a hothplug volume attachment pod.
+func IsAttachmentPod(pod *k8sv1.Pod) bool {
+	_, ok := pod.Annotations[virtv1.HotplugVolumeAttachmentAnnotation]
+	return ok
```

Comment:
Small suggestion:
```suggestion
	val, ok := pod.Annotations[virtv1.HotplugVolumeAttachmentAnnotation]
	return ok && strings.ToLower(val) == "true"
```
Otherwise the function will return `true` even if the annotation value is set to `false` (which is a bit confusing).